packages/scene-composer/src/components/three-fiber/EditorTransformControls.tsx

```diff
@@ -70,8 +78,10 @@ export function EditorTransformControls() {
         (selectedSceneNode.transformConstraint.snapToFloor !== true ||
           (selectedSceneNode.transformConstraint.snapToFloor === true && transformControlMode !== 'translate')) &&
         !(isLinearPlaneMotionIndicator(selectedSceneNode) && transformControlMode === 'scale');
+      // @ts-ignore
+      transformControls.flipY = isOverlayComponent && transformControlMode === 'translate';
```

Comment:
Blocker: Remove `ts-ignore`.

Reply:
I don't see the line in this PR. Can the comment be duplicated.

Reply:
duplicated